css/carliebydani.css

```diff
@@ -8,55 +8,88 @@
 
 @layer utilities {
     .wrapper {
-        @apply container
-            max-w-7xl
-            mx-auto
-            p-2;
+        @apply container max-w-7xl mx-auto p-2;
     }
 
     .body {
-        @apply bg-[#262626]
-        text-white;
+        font-family: 'Sofia Sans', sans-serif;
+        @apply bg-[#262626] text-white;
```

Comment:
Just extended the tailwind.config.js with our brand colors
use `bg-brand-lightgray` here